api/server/server.go

```diff
@@ -125,6 +131,30 @@ func NewServer(conf Config) (*Server, error) {
 	minerService := miner.NewService(mi)
 	slashingService := slashing.NewService(si)
 
+	grpcServer := grpc.NewServer(conf.GrpcServerOpts...)
+
+	wrappedServer := grpcweb.WrapServer(
+		grpcServer,
+		grpcweb.WithOriginFunc(func(origin string) bool {
+			return true
+		}),
+		grpcweb.WithWebsockets(true),
+		grpcweb.WithWebsocketOriginFunc(func(req *http.Request) bool {
+			return true
+		}),
+	)
+	handler := http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		if wrappedServer.IsGrpcWebRequest(r) ||
+			wrappedServer.IsAcceptableGrpcCorsRequest(r) ||
+			wrappedServer.IsGrpcWebSocketRequest(r) {
+			wrappedServer.ServeHTTP(w, r)
+		}
+	})
+	grpcWebProxy := &http.Server{
+		Addr:    conf.GrpcWebProxyAddress,
+		Handler: handler,
+	}
```

Comment:
Wrap the grpc service in the web proxy per https://github.com/improbable-eng/grpc-web/tree/master/go/grpcweb and consistent with how we do it in `go-threads`